test/workspaces/test_workspaces_create.py

```diff
@@ -6,8 +6,9 @@
 def test_workspaces_create(seam: Seam):
     r = "".join(random.choices(string.ascii_uppercase + string.digits, k=10))
     seam = Seam(
-        api_url=f"https://{r}.fakeseamconnect.seam.vc",
-        api_key="seam_at1_shorttoken_longtoken",
+        endpoint=f"https://{r}.fakeseamconnect.seam.vc",
+        personal_access_token="seam_at1_shorttoken_longtoken",
+        workspace_id="seed_workspace_1",
```

Comment:
This is interesting: Creating a workspace should not be allowed when using a PAT and a workspace ID.

Reply:
We should fix the [endpoint](https://github.com/seamapi/seam-connect/blob/46be41dedf1b7fe1aaf8f13a00326a0b994c2a4d/pages/api/public/workspaces/create.ts#L19) then. What type of auth should be allowed?

Reply:
Side node, I realize now that the name `SeamHttpWorkspaces` implies that operations should work for different workspaces, but it really needs to mean the client only provides operations that are "workspace less". Maybe we can think of a better name.

Reply:
Got it. So what approach should we take in Python SDK? Ignore `/workspaces/create` on Seam client and add `SeamMultiWorkspace` client that will support that endpoint and allow passing PAT without workspace?

Reply:
Yes 😀